docs/podman-build.1.md

```diff
@@ -317,23 +317,34 @@ podman build --security-opt label=level:s0:c100,c200 --cgroup-parent /path/to/cg
 
 podman build --volume /home/test:/myvol:ro,Z -t imageName .
 
-### Building an image using a URL
+### Building an image using a URL, Git repo, or archive
 
-  This will clone the specified GitHub repository from the URL and use it as context. The Dockerfile at the root of the repository is used as Dockerfile. This only works if the GitHub repository is a dedicated repository.
+  The build context directory can be specified as a URL to a Dockerfile, a Git repository, or URL to an archive. If the URL is a Dockerfile, it is downloaded to a temporary location and used as the context. When a Git repository is set as the URL, the repository is cloned locally to a temporary location and then used as the context. Lastly, if the URL is an archive, it is downloaded to a temporary location and extracted before being used as the context.
 
- `podman build github.com/scollier/purpletest`
+#### Building an image using a URL do a Dockerfile
```

Comment:
I'm not quite sure what you were trying to say in this title....

Reply:
Man, I reread this last night like five times and I still had a typo. Thanks for catching this. Fixed.